crossopt/assignment3/dictionary.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+
+class DictionaryNode:
```

Comment:
To clarify the API of this module, I think you can document DictionaryNode private (by prefixing). This helps the reader categorize who uses which objects. I understand in this exercice solution that no-one uses DictionaryNode outside of this module, am I right?